jobs/config.json

```diff
@@ -11234,72 +11234,6 @@
       "sig-scalability"
     ]
   },
-  "ci-kubernetes-kubemark-5-gce": {
```

Comment:
I think the reason for not removing this was to have smaller commit diff in case of some regression. But seems like this runs every 30m and kubemark-100 runs every 60m - so probably not worth keeping.

Reply:
And kubemark-500 every 1h5m too, so I don't think it's worth it.

Reply:
Ok, let's remove it.

Reply:
OK - so since this is the only one that doesn't seem controversial, I will move it to a separate PR.

Reply:
Extracted to: https://github.com/kubernetes/test-infra/pull/8584